src/setup/devnet/templates/docker/docker-bor-start.sh.njk

```diff
@@ -2,25 +2,22 @@
 
 set -x
 
-NODE_DIR=/root/.bor
-DATA_DIR=/root/.bor/data
-
 ADDRESSES=({% for s in obj.signerDumpData %}
   "{{ s.address }}"{% endfor %}
 )
 
 INDEX=$1;
 ADDRESS=${ADDRESSES[$INDEX]};
 
-NODE_DIR=/root/.bor
-DATA_DIR=/root/.bor/data
+BOR_DIR=/var/lib/bor
+BOR_DATA_DIR=/var/lib/bor/data
 
 docker compose run --service-ports -d --name bor$INDEX --entrypoint bash bor$INDEX -c "
 touch /root/logs/bor.log
 bor server \
-  --chain=$NODE_DIR/genesis.json \
+  --chain=$BOR_DIR/genesis.json \
```

Comment:
can we double check that - for docker and remote - the flags passed here with `cli` are exactly the same passed with the `config` setup? Just to be sure different node started with mixed `cli/config` setups run the same way

Reply:
Just checked and found some differences. Updated [here](https://github.com/maticnetwork/matic-cli/pull/215/commits/992ea1b1c6984f6986ae3b07545167036fd39064). Thank you!

Reference for [docker](https://github.com/maticnetwork/matic-cli/blob/master/src/setup/devnet/templates/docker/docker-bor-start.sh.njk) and [remote](https://github.com/maticnetwork/matic-cli/blob/master/src/setup/devnet/templates/remote/bor/bor-start.sh.njk).